ibm_catalog.json

```diff
@@ -310,6 +310,18 @@
                       "value": ".elser_model_2_linux-x86_64"
                     }
                   ]
+                },
+                {
+                  "key": "kibana_image_reference"
+                },
+                {
+                  "key": "existing_code_engine_project_id"
+                },
+                {
+                  "key": "enable_kibana_dashboard"
+                },
+                {
+                  "key": "elasticsearch_full_version"
```

Comment:
I actually added these in my [PR](https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/pull/363) which just got merged. You should rebase your branch with main and you will only need to add `kibana_image_reference` here

Reply:
Done.